### ISSUE TITLE
ctcloss = 0 (* 1 = 0 loss)

### ISSUE THREAD

**Issue by Ryansanity** [senlinuc/caffe_ocr#133]
请问作者我在训练的时候出现如下情况：
Test net output #0: acc = 0
I0806 15:36:56.926146 12768 solver.cpp:409]     Test net output #1: acc = 0
I0806 15:36:56.926173 12768 solver.cpp:409]     Test net output #2: ctcloss = 0 (* 1 = 0 loss)
I0806 15:36:56.974421 12768 solver.cpp:231] Iteration 0, loss = 0
I0806 15:36:56.974463 12768 solver.cpp:249]     Train net output #0: ctcloss = 0 (* 1 = 0 loss)
I0806 15:36:56.974516 12768 sgd_solver.cpp:106] Iteration 0, lr = 0.0001
I0806 15:37:10.347798 12768 solver.cpp:231] Iteration 200, loss = 0
I0806 15:37:10.347848 12768 solver.cpp:249]     Train net output #0: ctcloss = 0 (* 1 = 0 loss)
I0806 15:37:10.347867 12768 sgd_solver.cpp:106] Iteration 200, lr = 0.0001
I0806 15:37:23.756043 12768 solver.cpp:231] Iteration 400, loss = 0
I0806 15:37:23.756104 12768 solver.cpp:249]     Train net output #0: ctcloss = 0 (* 1 = 0 loss)
I0806 15:37:23.756115 12768 sgd_solver.cpp:106] Iteration 400, lr = 0.0001
I0806 15:37:37.147478 12768 solver.cpp:231] Iteration 600, loss = 0
I0806 15:37:37.147550 12768 solver.cpp:249]     Train net output #0: ctcloss = 0 (* 1 = 0 loss)
I0806 15:37:37.147568 12768 sgd_solver.cpp:106] Iteration 600, lr = 0.0001
I0806 15:37:50.544185 12768 solver.cpp:231] Iteration 800, loss = 0
I0806 15:37:50.544235 12768 solver.cpp:249]     Train net output #0: ctcloss = 0 (* 1 = 0 loss)
I0806 15:37:50.544246 12768 sgd_solver.cpp:106] Iteration 800, lr = 0.0001
请问是我训练数据标签的错误嘛？我的标签是不定长的，短的用10000补齐成了统一长度。求大神回复，谢谢。
